src/testcases/statom_short.test.ts

```diff
@@ -172,6 +173,48 @@ describe('StAtom / Short', () => {
         ).toEqual(1_000);
       });
     });
+
+    describe('receive tokenized share', () => {
+      let tokenizedShare: Coin;
+      let startingStatomBalance: number;
+      let exchangeRate: number;
+      beforeAll(async () => {
+        const balances = await testState.wrapper.queryClient.bank.allBalances(
+          (
+            await testState.wallets.a.getAccounts()
+          )[0].address,
+        );
+        tokenizedShare = balances.find((b) =>
+          b.denom.startsWith('wasmvaloper'),
+        );
+        startingStatomBalance = await querier.balance_statom(
+          testState.wallets.a,
+        );
+      });
+      test('receive tokenized share', async () => {
+        const res = await testState.lasset.receive_tokenized_share(
+          testState.wallets.a,
+          tokenizedShare,
+        );
+        expect(res.code).toEqual(0);
+      });
+      test('tokenized share must be spent', async () => {
+        const balances = await testState.wrapper.queryClient.bank.allBalances(
+          (
+            await testState.wallets.a.getAccounts()
+          )[0].address,
+        );
+        expect(
+          balances.find((b) => b.denom.startsWith('wasmvaloper')),
+        ).toBeFalsy();
+      });
+      test('statom balance must be increased', async () => {
```

Comment:
we would also like to check that after processing the tokenized share our delegation amount has increased

Reply:
added both steps